cmd/compute-domain-controller/computedomain.go

```diff
@@ -64,11 +64,13 @@ func NewComputeDomainManager(config *ManagerConfig) *ComputeDomainManager {
 	factory := nvinformers.NewSharedInformerFactory(config.clientsets.Nvidia, informerResyncPeriod)
 	informer := factory.Resource().V1beta1().ComputeDomains().Informer()
 
+	klog.Infof("Creating new ComputeDomainManager with config %+v", config)
```

Comment:
```suggestion
	klog.Infof("Creating new ComputeDomainManager with config %+v", config)
	
```

Reply:
This is actually not giving much valuable info 
`Creating new ComputeDomainManager with config &{driverName:compute-domain.nvidia.com driverNamespace:nvidia-dra-driver-gpu clientsets:
{Core:0x4000685a40 Nvidia:0x40003c8c50} workQueue:0x4000612020}`

may  be expand on the subfields?